client/my-sites/customer-home/style.scss

```
@@ -1,5 +1,9 @@
 @import './grid-mixins.scss';
 
+body.is-section-home {
+	background: white;
```

```suggestion
	background: var( --studio-white );
```

Maybe I'm overthinking it for such a basic colour, but maybe since we're dealing with colour schemes, we should use the colour scheme variable. It's hard to imagine `--studio-white` ever resolving to anything other than `#fff` though 🤷

---

Absolutely agree. We should use studio vars. If `—studio-white` ever changes to something else that's okay, the color system should be fluid!